third_party/2and3/tornado/httputil.pyi

```diff
@@ -29,19 +29,21 @@ class HTTPHeaders(Dict[Any, Any]):
     def __deepcopy__(self, memo_dict): ...
 
 class HTTPServerRequest:
-    method: Any
-    uri: Any
-    version: Any
-    headers: Any
-    body: Any
+    path: str
+    query: str
+    method: Optional[str]
```

Comment:
Github is giving a poor diff visualization here. `path` and `query` are new; `method` and below are just having their types refined from `Any`